chore: Docs: Cleanup Microsoft link

nschonni opened the pull request:
Ran the script from the docs-authoring-pack.
Looks like mostly helped remove locales

nschonni commented on docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md:
```diff
@@ -74,7 +74,7 @@ If you examine the "Sent a message over a channel" trace, you may see the follow
        <TraceData>
           <DataItem>
              <TraceRecord xmlns="http://schemas.microsoft.com/2004/10/E2ETraceEvent/TraceRecord" Severity="Information">
-                 <TraceIdentifier>http://msdn.microsoft.com/library/System.ServiceModel.Channels.MessageSent.aspx</TraceIdentifier>
```
Either URL is a 404, but I'm not sure if this is supposed to be HTTP like the schema URLs are

mairaw replied:
not sure either. @Lxiamail can you help? should this perhaps link to https://docs.microsoft.com/dotnet/framework/wcf/diagnostics/tracing/system-servicemodel-channels-messagesent?

mairaw replied:
Pinging @HongGit to see if she can help

nschonni replied:
@mairaw I've just removed this to unblock the PR

NextTurn commented on docs/fsharp/get-started/get-started-vscode.md:
```diff
@@ -10,7 +10,7 @@ You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the
 
 > [!NOTE]
-> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **OSX**, a simpler way to get started is to use the [command line tools](https://docs.microsoft.com/en-us/dotnet/fsharp/get-started/get-started-command-line).
+> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **OSX**, a simpler way to get started is to use the [command line tools](https://docs.microsoft.com/dotnet/fsharp/get-started/get-started-command-line).
```
```suggestion
> Ionide will generate .NET Framework F# projects, not dotnet core, which can have cross-platform compatibility issues. If you are running on **Linux** or **macOS**, a simpler way to get started is to use the [command line tools](get-started-command-line.md).
```

JRAlexander reviewed (approved):
LGTM for ML.NET related files. Thanks, @nschonni!

nschonni committed:
chore: Docs: Cleanup Microsoft link

Ran the script from the docs-authoring-pack.
Looks like mostly helped remove locales

BillWagner reviewed (approved):
This LGTM. I’ll :shipit: now.
